news/impressions.md

```diff
@@ -0,0 +1,21 @@
+title: Impressions of PyCon UK 2016
+date: 2016-09-20
+---
+If you were at PyCon UK 2016 and have written an article or weblog post about
+your experiences at the event, feel free to add it here at the end of the list
+below. To do that, simply make a pull request on the site's GitHub repository.
+
+This page is https://github.com/PyconUK/2016.pyconuk.org/blob/master/news/impressions.md - you can even edit it directly on GitHub and make your pull request there.
```

Comment:
If you wrap the URL in angle-brackets, then the Markdown parser auto-converts it to a link, i.e.:

```markdown
This page is <https://github.com/PyconUK/2016.pyconuk.org/blob/master/news/impressions.md> – you …
```